code/game/objects/items/weapons/pneumaticCannon.dm

```diff
@@ -88,9 +88,6 @@
 	if(!istype(user) && !target)
 		return
 	var/discharge = 0
-	if(is_in_gang(user, "Sleeping Carp"))
```

Comment:
xhuis coded pneumatic cannons and sleeping carp, and i guess cannons aren't real guns, you know the rest